systemtest/src/main/java/io/strimzi/systemtest/utils/TestKafkaVersion.java

```diff
@@ -28,20 +28,22 @@
 
     private static final Logger LOGGER = LogManager.getLogger(TestKafkaVersion.class);
     private static List<TestKafkaVersion> kafkaVersions;
+    private static List<TestKafkaVersion> supportedKafkaVersions;
+
 
     static {
         try {
-            List<TestKafkaVersion> tmpKafkaVersions = parseKafkaVersions(TestUtils.USER_PATH + "/../kafka-versions.yaml");
-            List<TestKafkaVersion> supportedKafkaVersions = tmpKafkaVersions.stream().filter(TestKafkaVersion::isSupported).collect(Collectors.toList());
+            kafkaVersions = parseKafkaVersions(TestUtils.USER_PATH + "/../kafka-versions.yaml");
+            supportedKafkaVersions = kafkaVersions.stream().filter(TestKafkaVersion::isSupported).collect(Collectors.toList());
+            Collections.sort(kafkaVersions);
             Collections.sort(supportedKafkaVersions);
 
-            kafkaVersions = supportedKafkaVersions;
-
-            if (kafkaVersions == null || kafkaVersions.size() == 0) {
+            if (supportedKafkaVersions == null || supportedKafkaVersions.size() == 0) {
                 throw new Exception("There is no one Kafka version supported inside " + TestUtils.USER_PATH + "/../kafka-versions.yaml file");
             }
 
-            LOGGER.info("These are following supported Kafka versions:\n{}", kafkaVersions.toString());
+            LOGGER.debug("These are following supported Kafka versions:\n{}", kafkaVersions.toString());
+            LOGGER.debug("These are following supported Kafka versions:\n{}", supportedKafkaVersions.toString());
```

Comment:
Is this supposed to be twice here? If yes, I guess the test should be different?